projects/sdk/src/lib/silo/Withdraw.ts

```diff
@@ -0,0 +1,69 @@
+import { ContractTransaction } from "ethers";
+import { Token } from "src/classes/Token";
+import { TokenValue } from "src/classes/TokenValue";
+import { BeanstalkSDK } from "../BeanstalkSDK";
+import { DepositCrate, Silo } from "../silo";
+import { sortCratesBySeason } from "../silo.utils";
```

Comment:
Why do we have both `silo.utils` and `./utils`? What's the difference between these two?